mip/driver_w5500.c

```diff
@@ -1,5 +1,6 @@
 #include "mip.h"
 
+#ifndef MG_ENABLE_DRIVER_IMXRT1020
```

Comment:
We don't need this, your driver should not clash with this one (no other driver does)

Reply:
This needs enums and phy functions to be prefixed. I have the draft for this but driver code needs be unique'd.

Reply:
STM32, STM32H, and TM4C, do not clash with this driver, and do not have prefixes